jx_sqlite/expressions.py

```diff
@@ -61,11 +61,15 @@ def to_sql(self, schema, not_null=False, boolean=False):
                 tempb = tempa.setdefault(schema.get_column_name(col), {})
                 tempb[json_type_to_sql_type[col.type]] = quote_column(col.es_column).sql
 
-    return wrap([
-        {"name": relative_field(cname, self.var), "sql": types, "nested_path": nested_path}
-        for nested_path, pairs in acc.items() for cname, types in pairs.items()
-    ])
-
+    cols = []
+    for nested_path, pairs in acc.items():
+        for cname, types in pairs.items():
+            if not startswith_field(cname, self.var):
```

Comment:
I was thinking of the time when `self.var` ( let's say `a`) is some table and `cname` (let's say `b`) is a column in it. `cname` here is named from perspective of the same table (`a`) and hence using relative field for it's name will lead to a wrong path (`..b`). This happens in deepops.